docs/pages/installation/generate_kubeconfig_eks.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+# Add user to k8s using service account, no RBAC (must create RBAC after this script)
+if [[ -z "$1" ]] || [[ -z "$2" ]]; then
+ echo "usage: $0 <service_account_name> <namespace>"
+ exit 1
+fi
+
+SERVICE_ACCOUNT_NAME=$1
+NAMESPACE="$2"
+KUBECFG_FILE_NAME="config-${SERVICE_ACCOUNT_NAME}-${NAMESPACE}.yaml"
+TARGET_FOLDER="."
+
+create_target_folder() {
+    echo -n "Creating target directory to hold files in ${TARGET_FOLDER}..."
+    mkdir -p "${TARGET_FOLDER}"
+    printf "done"
+}
+
+create_service_account() {
+    echo -e "\\nCreating a service account in ${NAMESPACE} namespace: ${SERVICE_ACCOUNT_NAME}"
+    kubectl create sa "${SERVICE_ACCOUNT_NAME}" --namespace "${NAMESPACE}"
+    kubectl create clusterrolebinding "${SERVICE_ACCOUNT_NAME}" --clusterrole=cluster-admin --serviceaccount=default:"${SERVICE_ACCOUNT_NAME}" --namespace "${NAMESPACE}"
+}
+
+get_secret_name_from_service_account() {
+    echo -e "\\nGetting secret of service account ${SERVICE_ACCOUNT_NAME} on ${NAMESPACE}"
+    SECRET_NAME=$(kubectl get sa "${SERVICE_ACCOUNT_NAME}" --namespace="${NAMESPACE}" -o json | jq -r .secrets[].name)
+    echo "Secret name: ${SECRET_NAME}"
+}
+
+extract_ca_crt_from_secret() {
+    echo -e -n "\\nExtracting ca.crt from secret..."
+    kubectl get secret --namespace "${NAMESPACE}" "${SECRET_NAME}" -o json | jq \
+    -r '.data["ca.crt"]' | base64 -D > "${TARGET_FOLDER}/ca.crt"
```

Comment:
On what type of OSes is the `-D` compatible with?

Reply:
I am not sure, Will check on that and get back to you lee.